File: sort.py
def sort(width: float, height: float, length: float, mass: float) -> str:
    volume = width * height * length
    is_bulky = volume >= 1_000_000 or max(width, height, length) >= 150
    is_heavy = mass >= 20
    
    if is_bulky and is_heavy:
        return "REJECTED"
    elif is_bulky or is_heavy:
        return "SPECIAL"
    else:
        return "STANDARD"


if __name__ == "__main__":
    width = float(input("Width (cm): "))
    height = float(input("Height (cm): "))
    length = float(input("Length (cm): "))
    mass = float(input("Mass (kg): "))
    print(sort(width, height, length, mass))

File: sort.ts
import * as readline from "readline";

function sort(width: number, height: number, length: number, mass: number): string {
    const volume = width * height * length;
    const isBulky = volume >= 1000000 || Math.max(width, height, length) >= 150;
    const isHeavy = mass >= 20;
    
    if (isBulky && isHeavy) {
        return "REJECTED";
    } else if (isBulky || isHeavy) {
        return "SPECIAL";
    } else {
        return "STANDARD";
    }
}

const rl = readline.createInterface({ input: process.stdin, output: process.stdout });

rl.question("Width (cm): ", (w) => {
    rl.question("Height (cm): ", (h) => {
        rl.question("Length (cm): ", (l) => {
            rl.question("Mass (kg): ", (m) => {
                console.log(sort(parseFloat(w), parseFloat(h), parseFloat(l), parseFloat(m)));
                rl.close();
            });
        });
    });
});
